app/controllers/change_requests_controller.rb

```diff
@@ -180,17 +180,34 @@ def persist_change(change_request)
       phone.update field_change_hash
     elsif change_request.is_a? AddressChangeRequest
       puts "AddressChangeRequest"
+      # action = change_request.field_changes["action"]
+      action = "" # to be removed for end-to-end testing
+      #puts "ACTION: #{action}"
+      # change_request.field_changes.each do |fc|
+      #   puts "#{fc.field_name} ==== #{fc.field_value}"
+      # end
       address = Address.find(change_request.object_id)
 
-      begin
-        a = Geokit::Geocoders::GoogleGeocoder.geocode address.address_1 + "," + address.city + "," + address.state_province
-        field_change_hash["latitude"] = a.latitude
-        field_change_hash["longitude"] = a.longitude
-      rescue => error
-        puts "google geocoding failed for address " + address.id.to_s + ": " + error.message
-      end
+      if action.downcase == "remove"
+        #address.destroy
+        #puts "ID: #{change_request.object_id.to_s}"
+        begin
+          address.destroy
+        rescue => error
+          puts error.message
```

Comment:
Just wondering, were you encountering errors trying to call `#destroy` on the Address? If so, what was the error?